File: app/src/main/java/ru/eva/oriokslive/models/miet/news/Item.java
package ru.eva.oriokslive.models.miet.news;

import org.simpleframework.xml.Element;
import org.simpleframework.xml.Root;

@Root(name = "item", strict = false)
public class Item {

    @Element(name = "title")
    private String title;
    @Element(name = "link")
    private String link;
    @Element(name = "description")
    private String description;
    @Element(name = "enclosure")
    private Enclosure enclosure;
    @Element(name = "pubDate")
    private String date;

    public String getTitle() {
        return title;
    }

    public String getLink() {
        return link;
    }

    public String getDescription() {
        return description;
    }

    public String getDate() {
        return date;
    }

    public Enclosure getEnclosure() {
        return enclosure;
    }
}

File: app/src/main/java/ru/eva/oriokslive/network/entity/news/Item.kt
package ru.eva.oriokslive.network.entity.news

import org.simpleframework.xml.Element
import org.simpleframework.xml.Root

@Root(name = "item", strict = false)
data class Item(
    @Element(name = "title")
    val title: String,
    @Element(name = "link")
    val link: String,
    @Element(name = "description")
    val description: String,
    @Element(name = "enclosure")
    val enclosure: Enclosure,
    @Element(name = "pubDate")
    val date: String,
)